### ISSUE TITLE
metrics and loss understanding

### ISSUE THREAD

**Issue by lucasvw** [jdb78/pytorch-forecasting#351]
Hi there,

I have more of conceptual understanding question with the library. I have always understood that metrics and loss are similar concepts but have an important difference:

- Metrics are the thing you are interested in as a data scientist. Generally computed during training after each epoch on the validation set for monitoring and thus don't influence the training itself. 
- Loss is the function that is used during training and on which sgd and backprop work to update the parameters.

I thought that metric aint used as loss function because they are not differentiable. In this library Im seeing that metrics and losses are used quite interchangeably. Would love to understand better how this works.

**Comment by jdb78** [comment]
There is not much difference between a metric and a loss. A loss is simply the (sometimes negative) metric that you minimize during training. You should evaluate your model against more than the loss metric to understand its weaknesses better.
